src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  let partner;
+
+  if (isMarried) {
```

Comment:
use `isMaried` in layout.

Reply:
why do you need to use the same logic in two places in one component?

Reply:
hurried, was not attentive...